client/landing/gutenboarding/index.tsx

```diff
@@ -93,20 +93,19 @@ window.AppBoot = async () => {
 	// Add accessible-focus listener.
 	accessibleFocus();
 
-	let locale = DEFAULT_LOCALE_SLUG;
+	let i18nLocaleData;
 	try {
 		const [ userLocale, { translatedChunks, ...localeData } ]: (
 			| string
 			| any
 		 )[] = await getLocale();
```

Comment:
No meaning, it's a nicety for breaking these onto multi-lines, similar to allowing additional commas:

```ts
type MultiLineUnion =
  | This // Leading | is nice, stable with less thrashing when items change
  | That
  | TheOtherThing

// Same as
type SingleLineUnion = This | That | TheOtherThing

// Analagous to
const arr = [
  'x',
  'y',
  'z', // trailing comma is nice, stable with less thrashing when array items change
]
```

I'd like to pull a few things in from the demo PR (https://github.com/Automattic/wp-calypso/pull/39627), one of them is that this can become a tuple I believe of the shape `[ string, {} ]`.

Reply:
Thanks, that's what I was guessing what it probably is -- a friend of the trailing comma 🙂